app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,69 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import * as yargs from 'yargs'
```

Comment:
Oh, i know what this is - you've modified the `package.json` but `npm-shrinkwrap.json` will override this - it needs to be added with an `npm install` to regenerate the shrinkwrap file.